agent/Cargo.toml

```
@@ -21,10 +24,15 @@ clap = { version = "4.5.16", features = ["derive"] }
 url = "2.5.2"
 tracing = "0.1.40"
 tracing-subscriber = "0.3.18"
+rand = "0.9.0"
 
 [build-dependencies]
 tonic-build = "0.12.0"
 
 [[bin]]
-name = "scheduler-mock"
-path= "src/mock/scheduler_mock.rs"
+name = "scheduler_mock"
+path = "scheduler_mock/main.rs"
+
+[[bin]]
+name = "agent"
```

can you rename it to sealci-agent ???

```suggestion
name = "sealci-agent"
```